build.sh

```diff
@@ -61,13 +63,25 @@ function parse_arguments() {
             shift; shift;
             ;;
             --offline)
-            BUILD_FLAGS="--embed-vsix:true"
+            if [ "$BUILD_FLAGS" == "" ]; then
+                BUILD_FLAGS+="--embed-vsix:true"
+            else
+                BUILD_FLAGS+=" --embed-vsix:true"
+            fi
             shift;
             ;;
             --skip-oci-image)
             SKIP_OCI_IMAGE="true"
             shift;
             ;;
+            --skip-digest-generation)
+            if [ "$BUILD_FLAGS" == "" ]; then
+                BUILD_FLAGS+="--skip-digest-generation:true"
```

Comment:
Hi, the if and else instruction seems to do the same stuff ?

Reply:
Done :+1: